wpiutil/src/main/native/cpp/FileLogger.cpp

```diff
@@ -28,13 +30,14 @@ FileLogger::FileLogger(std::string_view file,
       m_thread{[=, this] {
         char buf[4000];
         struct inotify_event ev;
-        int len = 0;
         lseek(m_fileHandle, 0, SEEK_END);
-        while ((len = read(m_inotifyHandle, &ev, sizeof(ev))) > 0) {
-          int bufLen = 0;
-          if ((bufLen = read(m_fileHandle, buf, sizeof(buf)) > 0)) {
+        while (read(m_inotifyHandle, &ev, sizeof(ev)) > 0) {
+          int bufLen = read(m_fileHandle, buf, sizeof(buf));
+          while (bufLen > 0) {
```

Comment:
For consistency, can do both while loops the same way.

Reply:
With or without the variable? The inotify loop does not need a variable, but file reading loop does to pass to the string_view.

Reply:
you can omit the variable for the inotify loop

Reply:
I think Peter means you can use `while ((bufLen = ...) > 0)`